streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -980,7 +980,12 @@ boolean maybeCommit() {
                 }
             }
 
-            lastCommitMs = now;
+            if (committed == -1) {
```

Comment:
What's this trace for?

Reply:
I thought it might be helpful to see that we are not committing some of what we've processed, because it is we can't commit during a rebalance. But I don't think it's absolutely necessary and can take it out if you don't think it adds much?